discord/commands/options.py

```diff
@@ -138,6 +148,10 @@ class Option:
         .. note::
 
             Does not validate the input value against the autocomplete results.
+    channel_types: Optional[List[:class:`discord.ChannelType`]]
+        A list of channel types that can be selected in this option.
+        Only applies to Options with an :attr:`input_type` of :class:`discord.abc.GuildChannel`.
```

Comment:
Maybe reference this as an `OptionType` instead

Reply:
Like this? 
```suggestion
        Only applies to Options with an :attr:`input_type` of :class:`discord.SlashCommandOptionType.channel`.
```

Reply:
Looks right

Reply:
Looks more left :p

Reply:
the line expanded to the right because more characters were added, therefore it is right not left